src/android/BackgroundGpsPlugin.java

```diff
@@ -84,6 +93,7 @@ public boolean execute(String action, JSONArray data, CallbackContext callbackCo
                 this.notificationTitle = data.getString(8);
                 this.notificationText = data.getString(9);
                 this.stopOnTerminate = data.getString(11);
+                this.postLocationsToServer = data.getBoolean(12);
```

Comment:
Yea, I think that would be clearer.  If we go with that option we should be sure to explicitly throw if a user provides other request like parameters (e.g. headers) wanted to make the least invasive change so I just added a property here.